src/VisualStudio/Core/Def/ServicesVisualStudio.csproj

```diff
@@ -81,36 +81,6 @@
     <Compile Include="..\..\..\Test\PdbUtilities\Shared\IMetadataImport.cs">
```

Comment:
Consider adding ` <!-- Copyright (c)  Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information. -->` at the top of the file.